consumer/init_consumer.py

```diff
@@ -1,18 +1,23 @@
-import sys, os
+import os
 #import boto3
 
 from consumer import Consumer
 from kafka_message.ttypes import KafkaMessage
 from thrift.TSerialization import deserialize
 from cassandra.cluster import Cluster
 
+
 def callback(msg):
+    try:
+        CASSANDRA_HOST = os.environ['CASSANDRA_HOST']
+    except KeyError as err:
+        print(err, "Please set CASSANDRA_HOST environment variable")
     kafka_message = KafkaMessage()
     deserialize(kafka_message, msg.value)
     print kafka_message
     
     #client = boto3.client("sns")
-    cluster = Cluster(['172.17.0.2']) #'35.162.115.250'
+    cluster = Cluster([CASSANDRA_HOST])
```

Comment:
cluster client is recreated every time when callback is called - please move client construction this out of callback and pass it into as argument